module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -943,9 +943,7 @@ public function editAction()
         }
 
         return $this->createViewModel(
-            [
-                'driver' => $driver, 'lists' => $lists, 'savedData' => $savedData
-            ]
+            compact('driver', 'lists', 'savedData', 'listFilter')
```

Comment:
Is there a reason for calling the view parameter listFilter instead of listID? Filter sounds like it could be something more than a single list ID.

Reply:
The template already expected a parameter called listFilter, which I think is a legacy of VuFind 1.x; I kept the name for backward compatibility... but since this parameter has been orphaned/broken for many years, it's probably better to change it to something more understandable. I will revise!